packages/client/hmi-client/src/types/workflow.ts

```diff
@@ -79,6 +79,12 @@ export interface WorkflowOutput<S> extends WorkflowPort {
 	operatorStatus?: OperatorStatus;
 	state?: S;
 	timestamp?: Date;
+	isSaved?: boolean;
```

Comment:
I think this better than adding it on `state` for each output 👍

Reply:
I would like to keep `WorkflowOutput` a bit cleaner if possible, also having a lot of optional typings is not a very good pattern when there are implicit dependencies amongst the optional fields.

Reply:
I think logically it makes sense to have the `isSaved` flag directly tied to the output seeing as it easily lets us know whether that specific output is saved or not.

Reply:
`isSaved` has now been removed. For output grouping, we're now checking against active project model ids.